tests/test_main.py

```diff
@@ -27,6 +28,28 @@ def test_main_succeeds(runner: CliRunner, segy_input: str, zarr_tmp: Path) -> No
     assert result.exit_code == 0
 
 
+@pytest.mark.dependency(depends=["test_main_succeeds"])
+def test_main_info_succeeds(runner: CliRunner, zarr_tmp: Path) -> None:
+    """It exits with a status code of zero."""
+    cli_args = ["info"]
+    cli_args.extend(["-i", str(zarr_tmp)])
+
+    result = runner.invoke(__main__.main, args=cli_args)
+    assert result.exit_code == 0
+
+
+@pytest.mark.dependency(depends=["test_main_succeeds"])
+def test_main_copy_succeeds(runner: CliRunner, zarr_tmp: Path, zarr_tmp2: Path) -> None:
+    """It exits with a status code of zero."""
+    cli_args = ["copy"]
+    cli_args.extend(["-i", str(zarr_tmp)])
+    cli_args.extend(["-o", str(zarr_tmp2)])
+
+    result = runner.invoke(__main__.main, args=cli_args)
+    print(f"copy returns {result}")
+    # assert result.exit_code == 0
```

Comment:
please remove these, no prints or commented code.
instead please add assertion to make sure it ran successfully.

Reply:
Removed